portable/release.sh

```diff
@@ -134,6 +144,12 @@ test -z "$include_pdbs" || {
 } ||
 die "Could not unpack .pdb files"
 
+# ARM64 Windows handling
+if test -n "$cross_compiling_arm64"
+then
+	cp -ar /arm64 "$SCRIPT_PATH/root"
```

Comment:
If we allow specifying an argument (like `--output <directory>`), we do not have to hard-code `/arm64` here.

I think I like that idea.

And if we go that route, the option should probably be renamed to `--include-arm64-artifacts=<directory>` or some such.

Reply:
Changed in https://github.com/git-for-windows/build-extra/pull/323/commits/6e5a2e74b5fbe829a7c93c82c0cab9487a8d5ad0. Can now run `./release.sh --include-arm64-artifacts=/arm64 --output=./arm64 2.30.0` (the `--output` option was already there and I simply use it to easily find back the generated `.7z.exe` file 😊)

Output:

```
$ ./release.sh --include-arm64-artifacts=/arm64 --output=./arm64 2.30.0
Including ARM64 artifacts from /arm64
7za is /usr/bin/7za
Creating archive

7-Zip (a) [32] 16.02 : Copyright (c) 1999-2016 Igor Pavlov : 2016-05-21
p7zip Version 16.02 (locale=nl_NL.UTF-8,Utf16=on,HugeFiles=on,32 bits,8 CPUs Intel(R) Core(TM) i7-9700 CPU @ 3.00GHz (906ED),ASM,AES-NI)

Scanning the drive:
92 folders, 6209 files, 708249591 bytes (676 MiB)

Creating archive: /tmp.7z

Items to compress: 6301


Files read from disk: 6203
Archive size: 48514387 bytes (47 MiB)
Everything is Ok
Success! You will find the new installer at "./arm64/PortableGit-2.30.0-arm64.7z.exe".
It is a self-extracting .7z archive.
```